### PR TITLE
update changelog

### PR DESCRIPTION
**Pull request by avpeery**
Update 2.8.1 changelog (version already updated in the cherry pick pr)

## Launch Checklist

 - [x] briefly describe the changes in this PR

 - [x] apply changelog label ('bug', 'feature', 'docs', etc) or use the label 'skip changelog'

### CODE REVIEW

**Review by SnailBones — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 2.8.1
+
+### Bug fixes 🐞
+* Fix the special bundle for CSP-restricted environments that broke in the v2.8.0 release.
```
Comment (SnailBones) [inline CHANGELOG.md:4]: Should we link to the PR?

Comment (SnailBones) [reply CHANGELOG.md:4]:
```suggestion
* Fix the special bundle for CSP-restricted environments that broke in the v2.8.0 release. ([#11739](https://github.com/mapbox/mapbox-gl-js/pull/11739))
```

Comment (avpeery) [reply CHANGELOG.md]: Added in thanks Aidan!

**Pull request merged**